.github/workflows/certora.yml

```diff
@@ -0,0 +1,56 @@
+name: Certora
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  verify:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+
+      matrix:
+        conf:
+          - DelegationEthreum
+          - ERC20Ethereum
+          - ImmutabilityEthreum
+          - MintAndBurnEthereum
+          - ReentrancyEthereum
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: recursive
+
+      - name: Install python
+        uses: actions/setup-python@v5
+        with:
+          python-version: ">=3.11"
+
+      - name: Install certora
+        run: pip install certora-cli
+
+      - name: Install solc (0.8.19)
+        run: |
+          wget https://github.com/ethereum/solidity/releases/download/v0.8.19/solc-static-linux
+          chmod +x solc-static-linux
+          sudo mv solc-static-linux /usr/local/bin/solc-0.8.19
+
+      - name: Install solc (0.8.27)
+        run: |
+          wget https://github.com/ethereum/solidity/releases/download/v0.8.27/solc-static-linux
+          chmod +x solc-static-linux
+          sudo mv solc-static-linux /usr/local/bin/solc-0.8.27
+
+      - name: Apply munging
+        run: make -C certora munged
+
```

Comment:
to remove to make the CI go through

Reply:
Fixed in https://github.com/morpho-org/morpho-token-upgradeable/pull/85/commits/bf1d4caac651a629435253001c0df22dfd8e16ac.